en/docs/references/choreo-connectors.md

```diff
@@ -0,0 +1,5 @@
+# Choreo Connectors
+
+Connectors in Choreo, exposed as API calls, allow you to connect to external services through your Choreo service. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
```

Comment:
Since this content is going to be common, I suppose it can cause confusion when we say "through your Choreo service" when this content is displayed in integration concepts.

Reply:
This will become a repetitive content as we have a similar thing under API Call topic in each md file